lib/model/frames/entity.js

```diff
@@ -50,7 +50,7 @@ class Entity extends Frame.define(
     const { uuid, label } = entityData.system;
     const { data } = entityData;
     return new Entity.Partial({ uuid, datasetId: dataset.id, id: oldEntity?.id }, {
-      def: new Entity.Def({ data, label, dataReceived: body.data || {} }),
+      def: new Entity.Def({ data, label, dataReceived: { ...body.data, label } || {} }),
```

Comment:
In the case of an update, `oldEntity` is specified, which means that even if no label was specified in the request body, `label` will exist (it will be taken from `oldEntity`). But in that case, I don't think we should include label in `dataReceived` (I don't think we should add a `label` property at all). How about something like:

```js
const dataReceived = { ...body.data };
if (body.label != null) dataReceived.label = body.label;
```